docs/first-order/test_2p_responses.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<blockquote>For our dataset, this criterion is rather stringent, only including 15 rois in the final selection. </blockquote><p> </p><p>For each of the inclusion criteria, the number stated in the markdown text and what is displayed in the notebook is different, I changed to keep consistent, but I'm not sure if a different dataset was supposed to be used instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>